### DISCUSSION TITLE
Add synced patterns / reusable block support

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Revisions#1118] [Feature Requests]
It will be good if our plugin can create revision on synced patterns / reusable block

**Comment by KratosGemini** [comment]
This would be very helpful. The main reason we don't really use this WordPress feature yet is because we rely on PublishPress Revisions functionality for our editing workflow.

**Comment by bwurst** [comment]
I was searching for a tool to plan publishing of content changes to specific dates (Don't need multi author workflows at all). I use reusable blocks heavily. Too sad that PublishPress Revisions cannot help me for now, it sounds very good in general but not for reusable blocks...
